src/Pages/ProductListing/ProductListing.jsx

```diff
@@ -0,0 +1,173 @@
+                                <input className="radio-btn" type="radio"/>
+                                <label className="lbl-price">High To Low</label>
+                            </div>
+                        </div>
+
+                        {/* <!-- SEPERATOR --> */}
+                        <hr className="solid separator"/>
+
+                        {/* <!-- RATINGS FILTER --> */}
+                        <p className="text-lg heading">Ratings</p>
+                        <div className="col-12 filters-container">
+                            {Object.keys(ratingsFilter).length?
+                                    Object.keys(ratingsFilter).map(key=>
+                                        <div className="ratings-filter">
+                                            <input className="radio-btn" type="radio"/>
+                                            <label className="lbl-price">{ratingsFilter[key]}</label>
+                                        </div>
+                                        )
+                                :null}   
+                        </div>
+                    </div>
+
+                    {/* <!-- RIGHT CONATINER - PRODUCTS --> */}
+                    <div className="col-9 right-container">
+                        <div className='right-container-header'>
+                            <p className="text-sm breadcrumb">{`HomeOTC > CategoriesDiabetesDevicesBlood > Glucose > Monitors`}</p>
+                            <button className="btn btn-sm btn-primary clear-filter-btn">Clear all filters</button>
+                        </div>
+                        {/* <!-- FILTER -CHIPS --> */}
+                        <div className="col-10 chips-container">
+                            <div className="chip-word-flex">
+                                <p className="chip-word">Accu-check<button className="cross-btn">X</button></p>
+                                <p className="chip-word">Rs 500<button className="cross-btn">X</button></p>
+                                <p className="chip-word">4 star rating plus<button className="cross-btn">X</button></p>
+                            </div>
+                            <input className="chip-search" placeholder="search here"></input>
+                        </div>
+
+                        {/* <!-- FILTERED products --> */}
+                        <div className="col-12 flex-container-row filtered-prdcts">
+                            {featuredProducts.length?
+                                    featuredProducts.map(item=>
+                                        <ProductCard 
+                                            key = {item.id}
+                                            id = {item.id}
+                                            img ={item.img}
+                                            prdctBadge = {item.prdctBadge}
+                                            title = {item.title}
```

Comment:
suggestion: instead of sending individual properties of the object, send the object directly like <ProductCard key={item.key} cardDetails={item} /> and destructure it in the ProductCard.

Reply:
done!